testcases/kernel/controllers/cgroup/cgroup_regression_test.sh

```diff
@@ -262,18 +262,26 @@ test_5()
 	subsys1=`tail -n 1 /proc/cgroups | awk '{ print $1 }'`
 	subsys2=`tail -n 2 /proc/cgroups | head -1 | awk '{ print $1 }'`
 
-	mount -t cgroup -o $subsys1,$subsys2 xxx cgroup/
+	# This test should fail
+	mount -t cgroup -o $subsys1,$subsys2 xxx cgroup/ 2> /dev/null
+	if [ $? -eq 0 ]; then
+		tst_resm TFAIL "mount $subsys1,$subsys2 should fail"
+		umount cgroup/
+		failed=1
+		return
+	fi
+
```

Comment:
Here you cannot really expect the system to have anything mounted.

LTP supports various system including these that do not mount cgroups automatically at boot. So we have to try to detect if anything has been mounted or not and act accordingly.

Also these tests would need more than this, ideally these should be rewritten to new test library to begin with...